ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -203,8 +203,8 @@ export class CanvasService {
       data = response.data
     } catch (error) {
       if (axios.isAxiosError(error) && error.response !== undefined) {
-        const { status, data } = error.response
-        if (status === HttpStatus.BAD_REQUEST && typeof data?.error === 'string' && data.error === 'invalid_request') {
+        const { status } = error.response
+        if (status === HttpStatus.BAD_REQUEST) {
```

Comment:
I just found a bug related to this. I am wondering if Canvas changed the response data, since the error string here now seems to be `invalid_grant`. Maybe it was always broken, but I am pretty sure we tested this. I'll have to investigate more this afternoon. The change here seems to fix the problem.

Reply:
See #397.

Reply:
Yeah, I was thinking we'd merge the other first, since it's a smaller change.

Reply:
#398 was merged.